data/config/uniswap-wrapper.rinkeby.usability.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  rinkeby: {
```

Comment:
I think you only need to export the config object instead of `rinkeby: config`

Reply:
The idea was to use it with `-c` option, as it's a extraordinary case, and for that I need to embed id in `network` object.